### ISSUE TITLE
Minutely probe registered in rails initializer doesn't run

### ISSUE THREAD

**Issue by fmfdias** [appsignal/appsignal-ruby#527]
Hi,

We have registered a minutely probe to track some custom metrics of our project using the class example in https://docs.appsignal.com/ruby/instrumentation/minutely-probes.html

However the information we track never ends up in appsignal and looking into the logs, it completely ignores the custom metric we created.
```
[2019-05-20T15:19:00 (process) #19883][DEBUG] Gathering minutely metrics with 2 probes
[2019-05-20T15:19:00 (process) #19883][DEBUG] Gathering minutely metrics with 'puma' probe
[2019-05-20T15:19:00 (process) #19883][DEBUG] Gathering minutely metrics with 'sidekiq' probe
```

After digging up the gem code, it seems the problem lays in the initialization process. When the code in the rails initializer run, the AppSignal::Minutely.start method was already invoked and thus any new registration is ignored.

A workaround that worked for us was to run AppSignal::Minutely.start again at the end of the initializer.

Need some help figuring out what could be wrong here or if the workaround is indeed the correct way of doing it.

Thank you!

**Comment by tombruijn** [comment]
Hi @fmfdias,

Looks like that broke in a recent release. I've created PR #528 with a fix for this issue. We'll release this in the next patch release.

Thanks for reporting!

**Comment by fmfdias** [comment]
Hi @tombruijn,

Thank you for such a quick reply and fix!
Will be waiting for the release.

Thanks!

**Comment by tombruijn** [comment]
@fmfdias the fix for this was released in Ruby gem 2.9.7

**Comment by fmfdias** [comment]
@tombruijn Thank you! Already using it!
